debian/stocks-server.install

```diff
@@ -0,0 +1,14 @@
+server/target/server.war /usr/lib/stocks-server/
+deploy-server/config/setup-ca /usr/lib/stocks-server/
+
+deploy-server/config/openssl*.cnf /usr/share/stocks-server/
+deploy-server/config/db-migration.sql /usr/share/stocks-server/
```

Comment:
I should have deleted this. It is definitely no longer needed.

Reply:
Should I delete these then?

Reply:
Yes, please.